siac/hostdbcmd.go

```diff
@@ -117,15 +120,30 @@ func hostdbcmd() {
 				uptimeRatio = float64(uptime) / float64(uptime+downtime)
 			}
 
+			scanHistStr := ""
+			displayScans := host.ScanHistory
+			if len(host.ScanHistory) > scanHistoryLen {
+				displayScans = host.ScanHistory[len(host.ScanHistory)-scanHistoryLen:]
+			}
+			for _, scan := range displayScans {
+				if scan.Success {
+					scanHistStr += "1"
+				} else {
+					scanHistStr += "0"
+				}
+			}
```

Comment:
what does this look like visually? Is it like:
```
11100110110
```
where each `1` represents a successful scan? If so, we can probably pick better sigils than 1 and 0. Maybe `‾`/`_` or something. Mostly they just need to be highly visually distinct.
Full example output would be nice.